### PR TITLE
Feature/admin dashboreds

### PR DESCRIPTION
**Pull request by Tommmy1018**

### CODE REVIEW

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -31,11 +39,176 @@ public function index()
...
+        $labels = [];
+        $data   = [];
+        $colors = ['#CAC2C7'];
+
+        for ($i=1; $i < 12; $i++){
+            $month = date('F',mktime(0,0,0,$i,1));
+            $count = 0;
+
+            foreach($users as $user){
+                if($user->month == $i)
```
Comment (Ninit29) [inline app/Http/Controllers/AdminController.php]: You need to enclose it with close and open parenthesis
```
if($user->month == $i) {
                $count = $user->count;
                break;
}
```

File: resources/views/admin/dashboard.blade.php [hunk 2]
```diff
@@ -5,82 +5,99 @@
 <!-- Chart.js for graph -->
 <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.bundle.js"></script>
 @vite(['resources/js/dashboard_graph.js'])
+<<<<<<< HEAD
 
-<div class="container w-50">
+@section('content')
+=======
```
Comment (Ninit29) [inline resources/views/admin/dashboard.blade.php]: Fix this extra characters

File: resources/views/admin/dashboard.blade.php [hunk 3]
```diff
@@ -5,82 +5,99 @@
 <!-- Chart.js for graph -->
 <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.bundle.js"></script>
 @vite(['resources/js/dashboard_graph.js'])
+<<<<<<< HEAD
```
Comment (Ninit29) [inline resources/views/admin/dashboard.blade.php]: Remove this

**Comment by Tommmy1018** [comment]
I cahanged whole cade on the page

**Review by Ditzler01 — APPROVED** [review]

**Pull request merged**